src/Uno.Extensions.Templates/content/unoapp-extensions/build/steps/ios.yml

```diff
@@ -0,0 +1,107 @@
+    ###################################################################
```

Comment:
We should discuss this - it is possible for us to update the plist based on the certs and provisioning profile. For example check out this script: https://github.com/builttoroam/pipeline_templates/blob/b7ff7ce2ce5e41ac0702e0ceb303b8130392dfb2/azure/stages/build-xamarin-ios.yml#L188